native/core/src/execution/jni_api.rs

```diff
@@ -61,6 +61,20 @@ use tokio::runtime::Runtime;
 use crate::execution::operators::ScanExec;
 use log::info;
 
+use once_cell::sync::Lazy;
+
+static TOKIO_RUNTIME: Lazy<Runtime> = Lazy::new(|| {
```

Comment:
Perhaps this should be once per session (a user could modify the config in the session or run different queries in a single job that might require independent configuration)?